src/main/java/org/gridsuite/notification/server/NotificationWebSocketHandler.java

```diff
@@ -95,12 +97,18 @@ public NotificationWebSocketHandler(ObjectMapper jacksonObjectMapper, @Value("${
             return res;
         }).map(m -> {
             try {
+                Map<String, Object> headers = new HashMap<>();
+                headers.put(HEADER_TIMESTAMP, m.getHeaders().get(HEADER_TIMESTAMP));
+                headers.put(HEADER_STUDY_NAME, m.getHeaders().get(HEADER_STUDY_NAME));
+                if (m.getHeaders().get(HEADER_UPDATE_TYPE) != null) {
```

Comment:
StudyName can be null

Reply:
I don't know how it could be null ...

Reply:
studyname is a filter like updateType
The client can create a websocket to be notified :
- by all changes for a given study and all update types : `/notify?studyName=toto`
- by all changes for all users and a given update type : `/notify?updateType=loadflow_status`
- by all changes for all users and all update types : `/notify`
- by only changes for a given study and an update type :` /notify?studyName=toto&updateType=loadflow_status`
Currently, only the first is used.
See unit tests with filters.

Reply:
Done